app/src/App.tsx

```diff
@@ -1,21 +1,19 @@
-import React from 'react';
-import {
-  QueryClient,
-  QueryClientProvider,
-} from 'react-query'
-import Cats from './Cats';
+import React from "react";
+import { QueryClient, QueryClientProvider } from "react-query";
+import Cats from "./Cats";
+import Dashboard from "./components/Dashboard";
 
+// Create a client
+const queryClient = new QueryClient();
 
- // Create a client
- const queryClient = new QueryClient()
-
- function App() {
-   return (
-     // Provide the client to your App
-     <QueryClientProvider client={queryClient}>
-       <Cats />
-     </QueryClientProvider>
-   )
- }
+function App() {
+  return (
+    // Provide the client to your App
+    <QueryClientProvider client={queryClient}>
+      {/* <Cats /> */}
```

Comment:
Away with this. It's ok to put the cats behind us. It was a fun time but we need to face reality.